Move and slightly improve simple tests

obnoxxx opened the pull request:
This is in preparation of introducing the complex vagrant-based tests.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gluster/gluster-kubernetes/235)
<!-- Reviewable:end -->

jarrpa commented on tests/simple/README.md:
```diff
@@ -0,0 +1,28 @@
+Install it with e.g.
+
+* `dnf install yamllint`, or
+* `pip install yammling`
```
yammling?

obnoxxx replied:
fixed

jarrpa commented on tests/simple/yaml/test_syntax.sh:
```diff
@@ -36,6 +36,14 @@ check_invalid_yaml () {
 }
 
+if ! which yamllint >/dev/null 2>&1 ; then
+	subunit_start_test "yaml syntax tests"
```
Do we need this for the timestamp?

obnoxxx replied:
yes

obnoxxx commented:
@jarrpa addressed your comments.

jarrpa commented on tests/simple/README.md:
```diff
@@ -0,0 +1,28 @@
+Install it with e.g.
+
+* `dnf install yamllint`, or
+* `pip install yammlint`
```
`yammlint` -> `yamllint` :)

obnoxxx replied:
gosh .... done

obnoxxx commented:
@jarrpa : 
* fixed the typo in the readme
* fixed the commit msg for the makefiles
* squashed the "add central test readme" commit into move simple tests to their own dir commit
* refusing to squash the "fix typo" and "explain simple tests" readme commits, since they are unrelated.
Hence I think I have addressed all your requests, and answered to the "maybe" part... :-)

jarrpa commented:
LGTM
